src/providers/object-provider.js

```diff
@@ -346,6 +346,19 @@ export default class YamcsObjectProvider {
                 }]
             }
         };
+
+        if (this.#isImage(obj)) {
+            obj.telemetry.values.push({
+                name: 'VIPER Image Thumbnail',
```

Comment:
Let's remove the reference to VIPER here.

Reply:
OK What do you want to call it? Just Image Thumbnail?